modules/nf-core/mageck/count/tests/main.nf.test

```diff
@@ -0,0 +1,71 @@
+// nf-core modules test mageck/count
+nextflow_process {
+
+    name "Test Process MAGECK_COUNT"
+    script "../main.nf"
+    process "MAGECK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mageck"
+    tag "mageck/count"
+
+    test("mus_musculus - fastq") {
```

Comment:
ah and we need to name the tests by input data, i thought it was mouse data but maybe its something else